.github/workflows/publish_gem.yml

```diff
@@ -0,0 +1,17 @@
+name: Publish Gem
+
+on:
+  push:
+    tags:
+      - v*
```

Comment:
Could we switch this to just push when the release is published? That feels more intentional to me.

```
on:
  release:
    types:
      - published
```

I'm also a little concerned about the behavior when pushing a new tag (or publishing a release) when the included Version string hasn't been changed on the *gem* - perhaps there's a way to fail the step before pushing if `"v#{gem_version}" != release_tag`?

Reply:
@nevinera I got this finally working properly, we don't have a new version to currently push to Rubygems. But you can see [here](https://github.com/usertesting/biscuit/runs/2739914470?check_suite_focus=true) that it attempted to push the current version and the rake aborted because that version already exists.